packages/travix-ui-kit/components/stepper/stepper.md

```diff
@@ -0,0 +1,15 @@
+Basic input:
+
+    <div>
```

Comment:
If you don't need to override styles anymore, you can remove these wrapping divs 🙂